CareWheels/www/app/views/individualStatus.html

```diff
@@ -181,8 +188,31 @@ <h2 class="title">Sensor Status, Trevor</h2>
       <div class="col"></div>
     </div>
   </ion-content>
+    </ion-side-menu-content>
+
+
+
+    <ion-side-menu side="right">
+      <ion-list>
+        <!-- make sure to have target point to parent, otherwise
+             it will open the page in the within the menu block.
+
+             also there are two external links on this menu page,
+             these links need to be handled as opening an external
+             app, or the devices native web browser.(cordova plug-in) -->
+
+        <ion-item href="groupStatus.html" target="_parent">CareWheels Group</ion-item>
+        <ion-item href="reminders.html" target="_parent">Reminders</ion-item>
+        <!-- https://carebank.carewheels.org/#login -->
+        <ion-item href="" target="_parent">CareBank</ion-item>
+        <!-- link to sense app -->
+        <ion-item href="" target="_parent">Sen.se</ion-item>
+        <ion-item href="login.html" target="_parent">Logout</ion-item>
```

Comment:
hard-coded the menu into your view, once i fix the menu i can replace this with one line